src/wallet/rpc/wallet.cpp

```diff
@@ -412,6 +412,7 @@ static RPCHelpMan unloadwallet()
                 {
                     {"wallet_name", RPCArg::Type::STR, RPCArg::DefaultHint{"the wallet name from the RPC endpoint"}, "The name of the wallet to unload. If provided both here and in the RPC endpoint, the two must be identical."},
                     {"load_on_startup", RPCArg::Type::BOOL, RPCArg::Optional::OMITTED_NAMED_ARG, "Save wallet name to persistent settings and load on startup. True to add wallet to startup list, false to remove, null to leave unchanged."},
+                    {"force", RPCArg::Type::BOOL, RPCArg::Optional::OMITTED_NAMED_ARG, "Force unload of wallet. This will abort any rescan in progress."},
```

Comment:
This is a positional parameter. See `simulaterawtransaction` for how to do named-only parameters in an options Object.

(Maybe move "load_on_startup" there too)